apps/web/pages/success.tsx

```diff
@@ -452,7 +458,7 @@ export default function Success(props: SuccessProps) {
                           </>
                         );
                       })}
-                    {bookingInfo?.smsReminderNumber && (
+                    {bookingInfo?.smsReminderNumber && hasSMSAttendeeAction && (
```

Comment:
Only show sms reminder number on success page if the event type of the booking actually has a workflow step with the SMS_ATTENDEE action